wiki/Usage_permissions/en.md

```diff
@@ -0,0 +1,427 @@
+---
+tags:
+  usage
+  permissions
+---
+
+# Usage permissions
+
+## Opt-in
+
+If you are an artist and wish to have your stance regarding whether osu! users may make use of your work known publicly or privately, please feel free to reach out to either [the osu! support team via email](mailto:support@ppy.sh) or any member of the [osu!team](/wiki/People/osu!team) with as much information as possible regarding what work you may want to allow or disallow.
+
+## Audio
+
+All tracks listed on any [Featured Artist listing](https://osu.ppy.sh/beatmaps/artists) are approved for use in the creation of osu! beatmaps. Tracks from a Featured Artist that do not appear in their listing are *not* licensed, and additional permissions may be required to use them.
+
+**Important note**: external media (such as gameplay videos submitted to sites like YouTube, livestreams) is subject to different licensing requirements that must be negotiated in most circumstances with the artist themselves. Some artists have broad permissions or allowances for external media and we have made best efforts to include this information where appropriate, but please ensure you double-check using primary sources like the artist's own sites/platforms.
+
+| Artist                                   | Status                               | Exceptions                                                  | Notes                                                                                                               |
```

Comment:
maybe easier to find what you're looking for if this table were split into two, one for all the "Allowed" artists and one for everybody else

Reply:
Or better split into three, one for FA, two for non-FA (allowed and disallowed). A citation of proofs (such as chat/e-mail screenshot) would be great too.

Reply:
I don't understand this. Alphabetical is best.

Case: If a user is unable to type a search term, at least then they can search a single list alphabetically, rather than three.

Reply:
Personally I would like to search non-FA artists randomly and listen to the song myself, since U like discovering songs in surprise. But you're right about this. If only there is a dynamic table to apply the filters in live...

Reply:
I would directly not mention featured artists that have no exception, and better put a paragraph saying that featured artists that are not on this list are always allowed.